### DISCUSSION TITLE
Maualy just get the win10 taskbar

### DISCUSSION THREAD

**Discussion by toretua** [valinet/ExplorerPatcher#3762] [General]
Hi!
If you do this with an exe file, it should be possible to create a step by step to procedure to do this manually.
This is for work, so we cant install random exe files on pc
My only end goal is to get the vertical taskbar :)

**Comment by pyrates999** [comment]
That is not possible.

please see this: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

**Comment by Amrsatrio** [comment]
Not possible entirely using registry configurations, exe/dlls are needed to achieve this. There's also Windhawk that has a mod to make the 11 taskbar vertical, but that's still an exe to get it installed.
